test/e2e/set-manifest-flags.ts

```diff
@@ -7,6 +8,25 @@ function parseIntOrUndefined(value: string | undefined): number | undefined {
   return value ? parseInt(value, 10) : undefined;
 }
 
+// Grab the tracesSampleRate from the git message if it's set
+function getTracesSampleRateFromGitMessage(): number | undefined {
+  const gitMessage = execSync(
+    `git show --format='%B' --no-patch "HEAD"`,
+  ).toString();
+
+  // Search gitMessage for `[flags.sentry.tracesSampleRate: 0.000 to 1.000]`
+  const tracesSampleRateMatch = gitMessage.match(
+    /\[flags\.sentry\.tracesSampleRate: (0*(\.\d+)?|1(\.0*)?)\]/u,
```

Comment:
Minor, is it worth abstracting this into a `getConfigFromGitMessage(flag: string): string` so this can be used in future?